### ISSUE TITLE
Name conflict with Client SDK

### ISSUE THREAD

**Issue by oshoemaker** [tradablebits/livekit-server-sdk-python#8]
Is it possible to rename this project to "livekitserver" or something similar so it does not conflict with the Python Client SDK?

**Comment by AlexJMohr** [comment]
Sure, I'm open to the idea. I'm not sure why you would want both the client and server installed in the same project though.

**Comment by oshoemaker** [comment]
There is a use case that having a headless client participating in the session can be beneficial. It allows a low overhead communication mechanism for controlling the session without the need for other external services. We may end up using the go SDK or Rust so this isn't high priority. I do have both working in the same client using a this SDK as a local module and seems to work well.

**Comment by davidzhao** [comment]
yeah this makes sense.. we are thinking about folding this package into a monorepo for both of our python SDKs.

The package names would be:
* Client SDK: livekit on PyPi. `livekit.rtc` in code
* Server SDK: livekit-api on PyPi. `livekit.api` in code

CC: @theomonnom

**Comment by v3ss0n** [comment]
i am having problems now , how to install this on pypi?

**Comment by theomonnom** [comment]
Hey, we’re going to release a Server SDK inside the client repo this week (mono repo, but different packages)
